src/battle_script_commands.c

```diff
@@ -802,7 +802,7 @@ void (* const gBattleScriptingCommandsTable[])(void) =
     Cmd_magnitudedamagecalculation,              //0xB9
     Cmd_jumpifnopursuitswitchdmg,                //0xBA
     Cmd_setsunny,                                //0xBB
-    Cmd_maxattackhalvehp,                        //0xBC
+    Cmd_halvehp,                        //0xBC
```

Comment:
tbh I'm not a huge fan of aligning the comments, but we should try to keep them aligned where there's already alignment.